src/utils/number.ts

```diff
@@ -1,5 +1,9 @@
 import { numberKeys } from "./key";
 
+export function isValidDecimal(value: number): boolean {
+  return Number(value) % 1 != 0;
```

Comment:
I think this can be simplified to `!Number.isInteger(value)`.

Reply:
Man you really do know all the nooks and crannies of the browser API!  Thanks! :)